include/wx/ribbon/buttonbar.h

```diff
@@ -156,6 +156,7 @@ class WXDLLIMPEXP_RIBBON wxRibbonButtonBar : public wxRibbonControl
                 wxRibbonButtonBarButtonState min_size_class);
     virtual void SetButtonMaxSizeClass(int button_id,
                 wxRibbonButtonBarButtonState max_size_class);
+    virtual void SetButtonBitmapSize(const wxSize& large_btn, const wxSize& small_btn);
```

Comment:
IMHO would be more consistent to use `size`, rather than `btn`, in the parameter names here:
```suggestion
    virtual void SetButtonBitmapSize(const wxSize& size_large, const wxSize& size_small);
```
(of course, in the rest of wx API we use `camelCase` rather than `snake_case`, so it's already not very consistent, but like this it's at least locally consistent)

Reply:
I am familiar with camel case, I have no problems to change it all by time.